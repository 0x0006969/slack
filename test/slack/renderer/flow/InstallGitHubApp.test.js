const { InstallGitHubApp } = require('../../../../lib/slack/renderer/flow');

describe('InstallGitHubApp message rendering', () => {
  test('works', async () => {
    const message = new InstallGitHubApp('https://github.com/apps/url-to-slack-app');
    expect(message.toJSON()).toMatchSnapshot();
  });
});
