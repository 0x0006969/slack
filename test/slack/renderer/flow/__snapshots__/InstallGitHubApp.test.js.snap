// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`InstallGitHubApp message rendering works 1`] = `
Object {
  "attachments": Array [
    Object {
      "actions": Array [
        Object {
          "style": "primary",
          "text": "Install GitHub App",
          "type": "button",
          "url": "https://github.com/apps/url-to-slack-app",
        },
      ],
      "color": "danger",
      "mrkdwn_in": Array [
        "text",
      ],
      "text": "Looks like the app isn't installed on your repository. Install it to proceed.",
    },
  ],
  "response_type": "ephemeral",
}
`;
